healsparse/healSparseMap.py

```diff
@@ -320,6 +348,11 @@ def _read_healsparse_file(filename, pixels=None):
            Name of the file to read.
         pixels : `list`, optional
            List of integer pixels from the coverage map
+           degrade_nside : `int`, optional
+           Degrade map to this nside on read.  None means leave as-is.
+           reduction : `str`, optional
+           Reduction method with degrade-on-read.
+           (mean, median, std, max, min, and, or, sum, prod).
```

Comment:
Is `wmean` missing intentionally?

Reply:
These docs should not have been there at all, I cleaned that up.  Thanks for noticing!